app/assets/javascripts/pages/scenario-form-page.js.coffee

```diff
@@ -0,0 +1,13 @@
+class @ScenarioFormPage
+  constructor:() ->
+    @format
+    @enabledSelect2
+
+  format: (icon) ->
+    originalOption = icon.element
+    '<i class="fa ' + $(originalOption).data('icon') + '"></i> ' + icon.text
+
+  enabledSelect2: () ->
+    $('.select2-fountawesome-icon').select2
+      width: '100%'
+      formatResult: format
```

Comment:
The edit page is working but not the new page. Do you mind giving me the new page re as well? I hacked around but didn't get anything yet.